nav2_core/include/nav2_core/exceptions.hpp

```diff
@@ -1,8 +1,7 @@
 /*
  * Software License Agreement (BSD License)
  *
- *  Copyright (c) 2017, Locus Robotics
```

Comment:
I don't think you should remove the original copyright. You can still add the new one however.

Reply:
Actually, looking at this in more detail, there's really no code here that was in the original file. The git history should be delete the old file, add the new file. That should be fine.

Also, are we making this BSD license?

Reply:
@mkhansen-intel I put the original copyright. It think I accidentally deleted it. Yes, most of the code in the original file had exceptions which were specific to DWB.
Now ```dwb::exceptions``` derives from this ```nav2_core::exceptions```.

BSD license was what the original file had. However, I don't know what it should be.

Reply:
There are about 2 or 3 lines that more or less match a different file that is copyright Locus Robotics. Please keep the copyright header as is and maybe add an Intel line as well.

eg. 
```
/*
/*
 * Software License Agreement (BSD License)
 * Software License Agreement (BSD License)
 *
 *
 *  Copyright (c) 2017, Locus Robotics
 *  Copyright (c) 2019, Intel Corporation
*   All rights reserved.
```
and then the rest of the text.